### ISSUE TITLE
Were you able to reproduce the 1.7% Top5 Accuracy Lost under VGG16 5X Channel Pruning?

### ISSUE THREAD

**Issue by mtyylx** [ReProduceByYQ/Channel_pruning_yq#2]
Hi Jason Yue,

Your code helps me A LOT (^_^) during understanding He Yihui's channel pruning paper. My question is, have you try fully prune VGG16 (from conv1_1 to conv4_3) using your implementation? Did you manage to achieve only 1.7% Top5 accuracy lost under 5X as the paper reported? 

I also re-implemented He Yihui's paper with only channel pruning, but when I prune VGG16 for 5X (pruning each layer's channel number to the exact same number of He Yihui's 5X caffemodel prototxt), and the Top5 accruacy loss is around 5%, and the paper repored 1.7% only.

Did you also experience this issue? Thanks.

**Comment by Johnson-yue** [comment]
@mtyylx I trained VGG16 using my own dataset , Not  PASCAL VOC or other Open Dataset. So  I don`t care about paper result.  For me, channel pruning is really useful, in my experiment（fine tune using my  dataset）, speed up 2x and without accuracy loss. 
I think, the Channel Pruning Paper have two part are both important:
      1).  pruning  network
      2). fine-tune with dataset
Maybe  your problem is fine-tune , or it just maybe.

**Comment by mtyylx** [comment]
Thank you! May your model be trained well ^_^

**Comment by Johnson-yue** [comment]
Mybe my dataset is simple  ~·~
